openpype/lib/path_tools.py

```diff
@@ -2,8 +2,12 @@
 import re
 import logging
 
+from openpype.api import Anatomy
```

Comment:
Use relative imports of lib functions inside lib. It is because of Python 2 hosts which may have different import order.
```suggestion
from .anatomy import Anatomy
```

Reply:
done